spyder/plugins/editor/plugin.py

```diff
@@ -1225,6 +1225,8 @@ def register_editorstack(self, editorstack):
         for method, setting in settings:
             getattr(editorstack, method)(self.get_option(setting))
         editorstack.set_help_enabled(CONF.get('help', 'connect/editor'))
+        editorstack.set_hover_hints_enabled(CONF.get('lsp-server',

```

Comment:
Should this be an option inside the editor or inside the lsp @ccordoba12 (either wy can work 🤷‍♂)?

Reply:
It's fine to leave it in the LSP confpage.